src/js/app.js

```diff
@@ -688,7 +688,9 @@ function notify(msg, info, pub) {
   function shouldNotify() {
     if (msg.time < loginTime) { return false; }
     if (info.selfAuthored) { return false; }
-    if (document.visibilityState === 'visible') { return false; }
+    //if (document.visibilityState === 'visible') { return false; }
+    if (activeChat == Object.keys(chats[pub].secrets)[0]) { return false; } // already looking at chat
```

Comment:
Notification should be shown even if the chat is active, but window not visible (minimized, in another tab etc)

Reply:
`activeChat === pub` would be the correct check — in case of a group chat the value of `pub` here is uuid, not public key. Maybe it should be renamed to `chatId` for clarity.